src/components/common/MyProfileComponent.js

```diff
@@ -23,11 +24,11 @@ function MyProfileComponent({ user, Role, Programs }) {
         </Space>
         <Space direction="vertical" className="card-child">
           <Text type="secondary">Role</Text>
-          <Text className="card-text">{user.role}</Text>
+          <Text className="card-text">{user.provider_role_id}</Text>
         </Space>
         <Space direction="vertical" className="card-child">
           <Text type="secondary">Programs</Text>
-          <Text className="card-text">{user.programs.length}</Text>
+          {/* <Text className="card-text">{user.programs.length}</Text> */}
```

Comment:
I assume this is a temporary commenting out. It is fine for now, but let's make sure to revisit this once we have the new DB working!